modules/client/examples/01-client/05-get-dao.ts

```diff
@@ -1,19 +1,26 @@
 /* MARKDOWN
-### Loading DAO details
-
-Handles retrieving DAO metadata using its address or ENS domain.
+### Get DAO details
 
+Gets a specific DAO's metadata using its address or ENS domain.
```

Comment:
This comes from the original documentation, but maybe since we call metadata to a field of the dao object "Gets a specific DAO's metadata" can be confusing

Reply:
Great catch! Fixed locally.